is_prime/script/src/bin/evm.rs

```diff
@@ -0,0 +1,127 @@
+//! An end-to-end example of using the SP1 SDK to generate a proof of a program that can have an
+//! EVM-Compatible proof generated which can be verified on-chain.
```

Comment:
Why are we going with evm compatible proofs?

Could we just do the simplest thing that SP1 allows? Or is there a good reason to go with something more complicated?

(I don't think our proof is evm-compatible? So it would be fair to require that of SP1.)